src/js/popup.js

```diff
@@ -54,9 +90,27 @@ async function showRelayPanel(tipPanelToShow) {
     currentPanel.textContent = `${panelId}`;
     upgradeButtonEl.textContent = panelStrings.upgradeButton;
     upgradeButtonIconEl.src = panelStrings.upgradeButtonIcon;
+
+    //Premium Panel
+    registerDomainImgEl.src = panelStrings.registerDomainImg;
+    registerDomainButtonEl.textContent = panelStrings.registerDomainButton;
+    registerDomainHeadlineEl.textContent = panelStrings.registerDomainHeadline;
+    aliasesUsedValEl.textContent = aliasesUsedVal;
+    emailsBlockedValEl.textContent = emailsBlockedVal;
+    emailsForwardedValEl.textContent = emailsForwardedVal;
+    aliasesUsedTextEl.textContent = panelStrings.aliasesUsedText;
+    emailsBlockedTextEl.textContent = panelStrings.emailsBlockedText;
+    emailsForwardedTextEl.textContent = panelStrings.emailsForwardedText;
```

Comment:
There's code in the [i18n.js](https://github.com/mozilla/fx-private-relay-add-on/blob/main/src/js/i18n.js) script that looks at every element with `class="i18n-content"`, and sets the `textContent` to the translated message using the message id value from the element's `data-i18n-message-id` attribute.

So, as long as all these elements have `class=i18n-content` and the right `data-i18n-message-id` attribute, you shouldn't need to assign the `textContent` value yourself.

Reply:
And once you remove these, you can probably remove all of the `premiumPanelWrapper.querySelector` lines above that are only getting the element in order to set the `textContent`.

Reply:
Oh nice, I was wondering why some tags had that `data-i18n-message-id` attribute while some didn't. This makes sense to me now, thank you!